skimage/draw/__init__.py

```diff
@@ -1,3 +1,11 @@
+"""Utilities for generating coordinates of different shapes and curves.
+
+Use the tools in this module to generate coordinates of basic geometric shapes
+curves such as ellipses or bezier curves. Additional tools allow the creation
+of masks from polygons or setting the color of manipulation.
```

Comment:
Should the end of this sentence be read as "... allow setting the color of manipulation?" First of all, what does it mean ("the color of the manipulation")? Since we wrote "... allow the creation ..." it would be more consistent to have something like "... allow the creation ... and the setup of ..." or "... allow creating ... and setting ..."

Reply:
yeah, makes sense